blocks/text.js

```diff
@@ -689,3 +689,90 @@ Blockly.Blocks['text_prompt'] = {
   mutationToDom: Blockly.Blocks['text_prompt_ext'].mutationToDom,
   domToMutation: Blockly.Blocks['text_prompt_ext'].domToMutation
 };
+
+Blockly.Blocks['text_count'] = {
+  /**
+   * Block for counting how many times one string appears within another string.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      'message0': Blockly.Msg.TEXT_COUNT_MESSAGE0,
+      'args0': [
+        {
+          'type': 'input_value',
```

Comment:
Although Google's JavaScript style guide recommends single-quoted strings over double quotes, this section is JSON.  JSON doesn't support single-quoted strings, so when this JSON snippet is later ported to Android and iOS, it will suddenly break (even though it works fine here).

Recommend changing these strings to use double quotes.  Here and other JSON blocks below.